demystifying_command_line/demystifying_command_line.md

```diff
@@ -0,0 +1,251 @@
+
+The operating system of your Windows or Mac computer has a highly complex graphical user interface, and so do most of the programs you might use as a consumer (things like Microsoft Word, Slack, Google Chrome, or Adobe Photoshop).  You use a mouse, not a keyboard, to click on pictures that represent programs in order to open them.  You use a mouse to click on words like "File" or "Window" to show a menu of options.  You might use a mouse to click and drag to select a portion of an image or a selection of words, and then delete or move your selection.
+
+If you're like many people, almost all of your computer interactions take place in a GUI.  Often, the people who make graphical user interfaces spend a lot of time figuring out the most frequent tasks people want to do, and they design the GUI to make those as easy as possible to do.
+
+What makes a GUI different than a CLI?  A CLI is completely text based and requires typed commands.  Let's learn more in the next few pages!
+
+## Mythology about the Command Line Interface
+
+If you've ever watched a Hollywood film in which the protagonist saves the day by typing cryptic commands into the computer (often with tense music in the background and very loud and fast typing), you've seen a depiction of a **command line interface**, or **CLI**.  These dramatic depictions often suggest a few false things about using a CLI: 
+
+* You have to be a computer genius to use the command line, 
+* The consequences of getting a command wrong are severe, and 
+* People who use the command line have all the things they need to type memorized (see also: they are geniuses).
+
+This means that many people who are new to the command line interface might find the idea of working in the CLI intimidating.  We hope we can explain the pros and cons of the command line so you can make a decision based on what's most useful to you, instead of on misconceptions about how difficult or dangerous working in the command line is.
+
+![A three-panel cartoon. A computer user, with username robm, types the command `sudo su` into the computer.  The computer responds with the warning "robm is not in the sudoers file.  This incident will be reported."  The  user asks another person, "Hey - who does sudo report these 'incidents' to?"  The other person replies, "You know, I've never checked." In the last panel, Santa Claus is seen adding a name to his "naughty" list.](https://imgs.xkcd.com/comics/incident.png"[Incident](https://xkcd.com/838/) comic by xkcd, [CC BY-NC 2.5](https://xkcd.com/license.html).")
```

Comment:
```suggestion
```
consider removing this comic -- I'm not sure if it will appeal to someone with absolutely no CLI knowledge or experience, or if all the unfamiliar terms in the text will feel confusing or alienating